packages/react-core/src/components/Form/FormFieldGroupHeader.tsx

```diff
@@ -2,16 +2,13 @@ import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Form/form';
 import { css } from '@patternfly/react-styles';
 
-// typo - rename to FormFieldGroupHeaderTitleTextObject during breaking change release
-export interface FormFiledGroupHeaderTitleTextObject {
+export interface FormFieldGroupHeaderTitleTextObject {
```

Comment:
This is exported,  do we have codemod issue for this?

Reply:
Thanks for the catch! now we do: https://github.com/patternfly/pf-codemods/issues/609